src/Sarif/Comparers/CodeFlowComparer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+/// <summary>
+/// Note: This comparer may not have all properties compared. Will be replaced by comprehensive
+/// comparer generated by JSchema as part of EqualityComparer in a planned comprehensive solution.
+/// Tracking by issue: https://github.com/microsoft/jschema/issues/141
+/// </summary>
+namespace Microsoft.CodeAnalysis.Sarif.Comparers
+{
+    internal class CodeFlowComparer : IComparer<CodeFlow>
```

Comment:
>internal [](http://example.com/codeflow?start=4&length=8)

All of these classes s/be public?

Reply:
was following the same pattern of Sarif SDK's EqualityComparers. The EqualityComparer class is internal and have a public reference in Sarif types, e.g.
public static IEqualityComparer<ArtifactLocation> ValueComparer => ArtifactLocationEqualityComparer.Instance;